lib/wallaby/experimental/selenium/webdriver_client.ex

```diff
@@ -235,7 +235,7 @@ defmodule Wallaby.Experimental.Selenium.WebdriverClient do
   """
   @spec take_screenshot(Session.t) :: binary
   def take_screenshot(session) do
-    with {:ok, resp}   <- request(:get, "#{session.url}/screenshot"),
+    with {:ok, resp}   <- request(:get, "#{session.session_url}/screenshot"),
```

Comment:
It looks like the typespec is incorrect here too. It should probably be
```elixir
@spec take_screenshot(Session.t | Element.t) :: binary
```

I didn't realize we could pass an element to this function, but I checked the tests and appears that we support it. 😄

Reply:
We allow either to be passed in. With phantom it would actually take a picture of the element. But with the other drivers its a bug.